src/pages/blog/[id].astro

```diff
@@ -94,6 +100,39 @@ const { Content, remarkPluginFrontmatter } = await render(post);
     }
   }
 
+  .main-wrapper {
+    display: grid;
+    grid-template-columns: minmax(25vw, 0.25fr) 1fr auto;
```

Comment:
This doesn't really behave nicely on larger resolutions. Alternatively it could be:
`grid-template-columns: minmax(25ch, 1fr) 2fr;`

More Context can be found in my comment on the PR timeline.